src/sema/statements.rs

```diff
@@ -1810,6 +1811,17 @@ fn try_catch(
     ns: &mut Namespace,
     diagnostics: &mut Diagnostics,
 ) -> Result<(Statement, bool), ()> {
+    if ns.target == Target::Solana {
+        diagnostics.push(Diagnostic::error(
+            *loc,
+            "The try-catch statement is not supported on Solana. Please, check \
```

Comment:
I think `check` is too colloquial (here the usage of `check` sounds like `check it out`). Something like `read` or `go to` sounds better to me.

Reply:
Fixed